src/TLuaInterpreter.cpp

```diff
@@ -4388,12 +4388,12 @@ int TLuaInterpreter::getRoomExits( lua_State *L )
 // TODO: Provide exit details:
 int TLuaInterpreter::getAllRoomEntrances( lua_State *L )
 {
-    int roomId;
+    int roomId = 0;
     if( ! lua_isnumber( L, 1 ) ) {
         lua_pushstring( L, tr( "getAllRoomEntrances: bad argument #1 type (room id as number expected, got %1!)" )
                         .arg( luaL_typename( L, 1 ) )
                         .toUtf8().constData());
-        lua_error( L );
+        return lua_error( L );
```

Comment:
Not wanting to be picky but that is `(int) luaL_error(lua_State *L, const char *fmt, ...);` from Lua 5.3!

OTOH the material on both does seem to be the same for 5.1 as well, other than the `lua_error` from 5.1 claims the message item used on the top of the stack can be any type but 5.3 does not make such an explicit pronouncement even though `luaL_error` for both versions states identically that a message is created from the supplied c printf format string and any additional passed arguments...